test/DistributionTest.js

```diff
@@ -0,0 +1,151 @@
+const DistributionContract = artifacts.require("./DistributionContract");
+const MyTokenn = artifacts.require("./MyTokenn");
+
+const {
+    ether,           // Big Number support
+    constants,    // Common constants, like the zero address and largest integers
+    expectEvent,  // Assertions for emitted events
+    expectRevert, // Assertions for transactions that should fail
+} = require('@openzeppelin/test-helpers');
+
+
+
+const chai = require("./setupchai.js");
+const BN = web3.utils.BN;
+const expect = chai.expect;
+
+contract("Distribution contract", async ([owner, acc2, acc3, acc4]) => {
+
+    let instanceToken;
+    let instanceDistribution;
+
+    before(async () => {
+        instanceToken = await MyTokenn.deployed();
+        instanceDistribution = await DistributionContract.deployed();
+    });
+
+    it("Mint tokens", async () => {
+        expect(await instanceToken.balanceOf(owner)).to.be.bignumber.equal(ether('1000'));
+    });
+
+    it("Transfer deposit tokens", async () => {
+        await instanceToken.approve(DistributionContract.address, ether('400'));
+        await instanceDistribution.deposit(ether('400'));
+        expect(await instanceToken.balanceOf(DistributionContract.address)).to.be.bignumber.equal(ether('400'));
+        expect(await instanceToken.balanceOf(owner)).to.be.bignumber.equal(ether('600'));
+    });
+
+    describe("Add beneficiaries function , fill array", async () => {
+
+        it("Add beneficiaries - fail , address = 0", async () => {
```

Comment:
you can use next groupings:
describe("addBeneficiaries" => {
 describe("fail if ..." =>{
 it..
 it..
}
describe("successs if ..." =>{
 it ..
 it ..
}
})
to more clarify